src/main/java/gui/TimelinesGridPane.java

```diff
@@ -94,13 +93,20 @@ public void removeCameraShotBlock(CameraShotBlock block) {
      */
     private void addPanes() {
         panes = new ArrayList<>();
+        int c;
         for (int i = 0; i < numberOfTimelines; i++) {
+            c = 1;
             for (int j = 0; j < numberOfCounts; j++) {
                 SnappingPane pane = new SnappingPane(j, i, 200, countHeight);
                 this.add(pane, i, j);
                 panes.add(pane);
-                pane.setStyle("-fx-border-color: gray;"
-                        + "-fx-border-width: 0.1px 1px 0.1px 1px;");;
+                if (c > 4) {
```

Comment:
Maybe add a little comment about what style pattern the grid gets from this loop